samples/src/main/java/ua/net/tokar/json/filters/sample/Group.java

```diff
@@ -0,0 +1,14 @@
+package ua.net.tokar.json.filters.sample;
+
+public class Group {
+    public final Integer id;
+    public final String title;
+    public final Link users;
```

Comment:
private?

Reply:
they are final. Also, it will be required to provide getters or more verbose configuration to expose the properties to JSON